CMakeLists.txt

```diff
@@ -103,384 +16,312 @@ SET(PA_COMMON_SOURCES
+        endif()
+      endif()
+
+      mark_as_advanced(OSS_INCLUDE_DIR)
+      if(OSS_INCLUDE_DIR)
+        message(STATUS "Found OSS: ${OSS_INCLUDE_DIR}")
+        target_include_directories(portaudio PRIVATE "${OSS_INCLUDE_DIR}")
+        target_sources(portaudio PRIVATE src/hostapi/oss/pa_unix_oss.c)
+      else()
+        message(WARNING "OSS NOT found")
+      endif()
+    endif()
+  endif()
+endif()
+
+#
+# Installation
+#
+
+include(CMakePackageConfigHelpers)
+include(GNUInstallDirs)
+
+if(NOT CMAKE_FRAMEWORK)
+  install(FILES README.md DESTINATION "${CMAKE_INSTALL_DOCDIR}/portaudio")
+  install(FILES LICENSE.txt DESTINATION "${CMAKE_INSTALL_DOCDIR}/portaudio")
+
+  configure_file(cmake/portaudio-2.0.pc.in "${CMAKE_CURRENT_BINARY_DIR}/portaudio-2.0.pc" @ONLY)
+  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/portaudio-2.0.pc"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+
+  configure_package_config_file(cmake/portaudioConfig.cmake.in
+    "${CMAKE_CURRENT_BINARY_DIR}/cmake/portaudio/portaudioConfig.cmake"
+    INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio"
+    NO_CHECK_REQUIRED_COMPONENTS_MACRO
+  )
+  write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/cmake/portaudio/portaudioConfigVersion.cmake"
+    VERSION "${PORTAUDIO_VERSION}"
+    COMPATIBILITY SameMajorVersion
+  )
+  install(EXPORT portaudio-targets FILE "portaudioTargets.cmake"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio")
+  export(TARGETS portaudio
+    FILE "${CMAKE_CURRENT_BINARY_DIR}/cmake/portaudio/portaudioTargets.cmake")
+  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/cmake/portaudio/portaudioConfig.cmake"
+                "${CMAKE_CURRENT_BINARY_DIR}/cmake/portaudio/portaudioConfigVersion.cmake"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio")
+
+  if(NOT TARGET uninstall)
+    configure_file(
+      "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_uninstall.cmake.in"
       "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
       IMMEDIATE @ONLY)
-    ADD_CUSTOM_TARGET(uninstall
-      COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
-  ENDIF()
-ENDIF()
-
-# Prepared for inclusion of test files
-OPTION(PA_BUILD_TESTS "Include test projects" OFF)
-IF(PA_BUILD_TESTS)
-  SUBDIRS(test)
-ENDIF()
-
-# Prepared for inclusion of test files
-OPTION(PA_BUILD_EXAMPLES "Include example projects" OFF)
-IF(PA_BUILD_EXAMPLES)
-  SUBDIRS(examples)
-ENDIF()
+    add_custom_target(uninstall
+      COMMAND ${CMAKE_COMMAND} -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
+  endif()
+endif()
+
+set_target_properties(portaudio PROPERTIES
+  PUBLIC_HEADER "${PORTAUDIO_PUBLIC_HEADERS}"
+  MACOSX_FRAMEWORK_IDENTIFIER com.portaudio
+  FRAMEWORK_VERSION A
+  VERSION ${PROJECT_VERSION}
+  SOVERSION ${PROJECT_VERSION}
+  WINDOWS_EXPORT_ALL_SYMBOLS TRUE
+)
+install(TARGETS portaudio
+  EXPORT portaudio-targets
+  PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/portaudio"
```

Comment:
I think this requires `#include <portaudio/portaudio.h>` instead of `#include <portaudio.h>`.

Reply:
3627b03b7ac2b105ae8815f7e5d7aee4ecca67d5